nixos/tests/keter.nix

```diff
@@ -0,0 +1,32 @@
+import ./make-test-python.nix ({ pkgs, ... }: {
+  name = "keter";
+  meta = with pkgs.lib.maintainers; {
+    maintainers = [ jappie ];
```

Comment:
You need to add yourself to `/maintainers/maintainer-list.nix`.

Reply:
I'm already in on line 5691, I maintain cut-the-crap as well

Reply:
Huh, I searched for `japple` instead of `jappie` (^: